### PR TITLE
Accept a comma-separated list of ports to forward

### PR DESCRIPTION
**Pull request by jkutner**
Supersedes #15

### CODE REVIEW

**Comment by beanieboi** [comment]
@jkutner Do we need backend changes for this?

**Comment by jclem** [comment]
Should we remove the `localPort` option and its documentation?

**Comment by jkutner** [comment]
@beanieboi nope, no backend changes. I added you just to get another set of eyes on it.

**Comment by jkutner** [comment]
@jclem oh hmm, i didn't notice that `localPort` was affected. I need to think on that.

**Comment by jclem** [comment]
@jkutner We could make them mutually exclusive and just error out if both formats are used (the local port option and `port:port` or a list of ports).

File: commands/port.js [hunk 1]
```diff
@@ -31,24 +36,33 @@ module.exports = function(topic, command) {
 
 function * run(context, heroku) {
   yield exec.initFeature(context, heroku, function *(configVars) {
-    let remotePort = context.args.port;
-    let localPort = context.flags.localPort || remotePort;
```
Comment (jkutner) [inline commands/port.js]: @RasPhilCo is there a pattern for deprecating an arg? Like can we still accept it, but remove it from the auto-gen documentation?

Comment (RasPhilCo) [reply commands/port.js]: Make the deprecated arg hidden via `hidden: true`.

**Comment by jclem** [comment]
🎉 Thanks @jkutner! When-ish will this be available in the CLI?

**Comment by axelson** [comment]
@jclem This is now available in the heroku command line, at least when on ```
$ heroku version
heroku/7.33.1 darwin-x64 node-v11.14.0
```

By the way do you have a link to how to use `heroku ps:forward` to connect to Elixir/Erlang's observer? I can't find the link anymore (or maybe it was just on slack...)

**Pull request merged**